samples/cpp/build_samples_msvc.ps1

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Arguments parsing
+param (
+    [string]$BuildDirectory = "",
+    [string]$InstallDirectory = "",
+    [switch]$Help,
+    [switch]$h
+)
+
+$SourceDirectory = Split-Path $MyInvocation.MyCommand.Path
+$SamplesType = (Get-Item $SourceDirectory).Name
+$BuildDirectory = if ($BuildDirectory) {$BuildDirectory} else {"$Env:USERPROFILE/Documents/Intel/OpenVINO/openvino_${SamplesType}_samples_build"}
```

Comment:
Technically, if we remove the `$Env:USERPROFILE` part and use some other default build directory, this script could be used not only on Win but on other OSes with PowerShell installed.

Reply:
I guess it could be done via a separate ticket+PR later as it might require a bit more changes.